### PR TITLE
Scaffolding

### PR DESCRIPTION
**Pull request by martijnvandermarel**
@TimBuckers @Ishadijcks 
Removed Clion project, added folders for GES and PDB slave and readmes per folder

### CODE REVIEW

File: ges/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#GES
```
Comment (TimBuckers) [inline ges/README.md]: `# GES`

File: pdb-slave/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#PDB-slave
```
Comment (TimBuckers) [inline pdb-slave/README.md]: # PDB-slave

File: ges/README.md (hunk 1 above)
Comment (Ishadijcks) [inline ges/README.md]:
```suggestion
# GES
```

File: pdb-slave/README.md (hunk 2 above)
Comment (Ishadijcks) [inline pdb-slave/README.md]:
```suggestion
# PDB-slave
```

**Pull request merged**